db/migrate/20160226221827_rename_solutions_to_collaborator_solutions.rb

```diff
@@ -0,0 +1,23 @@
+class RenameSolutionsToCollaboratorSolutions < ActiveRecord::Migration
+  def up
+    # Transfer any solution Attachments to their Exercises
+    Attachment.where(parent_type: 'Solution').preload(:parent).find_each do |attachment|
+      attachment.update_attribute(:parent, attachment.parent.question.exercise)
+    end
```

Comment:
Do we need other changes in the importer to make future imported attachments go to the Exercise instead of the Solution?